custom-connectors/AzureADPhoneMethods/Readme.md

```diff
@@ -0,0 +1,67 @@
+
+## Azure AD phone methods Connector
+With this connector you can manage phone authentication methods to Azure Active Directory. This methods can be used for Azure MFA and Self Service Password Reset
+
+
+
+## Prerequisites
+You will need the following to proceed:
+* A Microsoft Power Apps or Power Automate plan with custom connector feature
+
+## Building the connector 
+Since the Azure AD authentication methods APIs are secured by Azure Active Directory (AD), we first need to set up a few things in Azure AD so that our connectors can securely access the phone methods. After that is completed, you can create and test the sample connector.
+
+### Set up an Azure AD application for your custom connector
+We first need to register our connector as an application in Azure AD.  This will allow the connector to identify itself to Azure AD so that it can ask for permissions to access Graph API data on behalf of the end user.  You can read more about this [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/authentication-scenarios) and follow the steps below:
+
+1. Create an Azure AD application
+This Azure AD application will be used to identify the connector to the Microsoft Graph API.  This can be done using [Azure Portal] (https://portal.azure.com), by following the steps [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).  Once created, note down the value of Application (Client) ID.  You will need this later.
```

Comment:
There's a space between `[Azure Portal]` and the link to it, causing it not to be parsed as intended.